### ISSUE TITLE
Java client lib: improve packaging

### ISSUE THREAD

**Issue by jerith666** [CarnegieLearningWeb/educational-experiment-service-client-libs#15]
The package names in this library are too generic:

```
$ git grep -h ^package -- java/ | sort -u
package example;
package interfaces;
package responsebeans;
package utils;
```

I think we need to use the standard reverse-domain-name namespacing to make them unique.  I'm not sure how tightly we want to tie this kind of technical detail to Carnegie Learning, versus, say, the Gates or Schmidt foundations, etc.  Possibilities include:

```
package org.upgrade.interfaces; // upgrade.org is, believe it or not, just a domain squatter
package com.carnegielearning.dev.upgrade.interfaces;
package org.tsffoundation.upgrade.interfaces;
package org.gatesfoundation.upgrade.interfaces;
```

cc @SritterCL for thoughts.

**Comment by dhavalocked** [comment]
What should be the final package name? `com.carnegielearning.dev.upgrade.interfaces` is fine?

**Comment by jerith666** [comment]
We need a similar change in the maven `<groupId/>`

**Comment by jerith666** [comment]
@dhavalocked I'm going to defer to @SritterCL on what organization he wants represented in the package names here.

**Comment by dhavalocked** [comment]
Hey @jerith666  @SritterCL  let me know when you decide the name. So that we can close this.

**Comment by jerith666** [comment]
Per @SritterCL, please use `org.upgradeplatform` as the maven groupId and the prefix for all Java package names.

Please use the appropriate analog for the npm package as well.

**Comment by dhavalocked** [comment]
fixed in #34

**Comment by jerith666** [comment]
#34 does address this, thanks!
